cleverhans/model.py

```diff
@@ -68,8 +68,12 @@ def get_params(self):
         Provides access to the model's parameters.
         :return: A list of all Variables defining the model parameters.
         """
-        scope_vars = tf.get_collection(tf.GraphKeys.TRAINABLE_VARIABLES,
-                                       self.scope)
+        if tf.executing_eagerly() == False:
```

Comment:
if tf.executing_eagerly() == False:

Simpler to write:
if not tf.executing_eagerly():

Or even better:
if tf.executing_eagerly():
   # raise
else:
  scope_vars